modules/tensor_mechanics/src/materials/abaqus/AbaqusUmatMaterial.C

```diff
@@ -214,21 +218,19 @@ AbaqusUmatMaterial::computeQpStress()
         _aqDFGRD0.data(),
         _aqDFGRD1.data(),
         &_aqNOEL,
-        &_aqNPT,
+        &_qp,
         &_aqLAYER,
         &_aqKSPT,
-        &_aqKSTEP,
+        &_t_step,
         &_aqKINC);
 
-  // Energy outputs
-  _elastic_strain_energy[_qp] = _aqSSE;
-  _plastic_dissipation[_qp] = _aqSPD;
-  _creep_dissipation[_qp] = _aqSCD;
-
   // Update state variables
   for (unsigned int i = 0; i < _num_state_vars; ++i)
     _state_var[_qp][i] = _aqSTATEV[i];
 
+  // determine the material timestep
+  _material_timestep[_qp] = _aqPNEWDT < 1.0 ? _aqPNEWDT * _dt : std::numeric_limits<Real>::max();
```

Comment:
Is this being tested?

Reply:
This is straight from the UMAT specifications. To test it we'd need a UMAT that changes the `PNEWDT` value.